docs/tutorials/super-for-jq.md

```diff
@@ -1,46 +1,59 @@
-of the [SuperPipe language](../language/_index.md) or [`super` command-line usage](../commands/super.md).
+SuperSQL’s pipeline extensions and shortcuts provide a flexible and powerful way for SQL
+lovers to query their JSON data while leveraging their existing skills.
+However, users who've traditionally wrangled their JSON with other tools such
+as [`jq`](https://stedolan.github.io/jq/) will find `super` equally powerful
+even if they don't know SQL or just prefer to work primarily in shortcuts. This tour
+walks through a number of examples using [`super`](../commands/super.md) at
+the command line with `jq` as a
+reference point.
 
-We'll start with some simple one-liners on the command line where we feed
-some data to `super` with `echo` and specify `-` for `super` input to indicate
-that standard input should be used, e.g.,
+We'll start with some simple one-liners where we feed some data to `super`
+with `echo` and specify `-` as input to indicate that standard input
+should be used, e.g.,
 ```
 echo '"hello, world"' | super -
 ```
 Then, toward the end of the tour, we'll experiment with some real-world GitHub data
 pulled from the GitHub API.
 
+Of course, if SQL is your preference, you can write many of the examples shown
+using the SQL equivalent, e.g.,
+```
+super -c "SELECT VALUE 'hello, world'"
+```
+
+or a mix of SQL and pipeline extensions as suits your preferences. However,
+to make this tutorial relevant to `jq` users, we'll lean heavily on SuperSQL's
+use of pipes and shortcuts.
+
 If you want to follow along on the command line,
 just make sure the `super` command [is installed](../install.md)
-as well as [`jq`](https://stedolan.github.io/jq/).
+as well as [`jq`](https://jqlang.org/download/).
 
 ## But JSON
 
 While `super` is based on a new type of [data model](../formats/data-model.md),
-Zed just so happens to be a superset of JSON.
+its human-readable format [Super JSON (JSUP)](../formats/jsup.md) just so
+happens to be a superset of JSON.
 
-So if all you ever use `zq` for is manipulating JSON data,
-it can serve you well as a handy, go-to tool.  In this way, `zq` is kind of
+So if all you ever use `super` for is manipulating JSON data,
+it can serve you well as a handy, go-to tool.  In this way, `super` is kind of
 like `jq`.  As you probably know, `jq`
 is a popular command-line tool for taking a sequence of JSON values as input,
 doing interesting things on that input, and emitting results, of course, as JSON.
 
 `jq` is awesome and powerful, but its syntax and computational model can
-sometimes be daunting and difficult.  We tried to make `zq` really easy and intuitive,
-and it is usually faster, sometimes [much faster](../commands/super.md#performance),
+sometimes be daunting and difficult.  We tried to make `super` really easy and intuitive,
+and it is usually faster, sometimes [much faster](https://github.com/brimdata/super/tree/main/performance),
```

Comment:
The ["performance" section of the `super` command doc](https://superdb.org/docs/commands/super/#performance) this was previously linking to has already (appropriately!) been morphed into a perf comparison vs. SQL-centric solutions, not `jq`. Therefore I'm instead pointing to the place in the repo that still has some `jq` perf comparisons. That said, I also recognize that same table of perf results includes `zeek-cut` stuff that's probably due to be dropped. If this PR passes muster, as a follow-on task I'll overhaul that perf table and the underlying script that generates the results.